<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryCardEditorDialog</class>
 <widget class="QDialog" name="MemoryCardEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory Card Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTableWidget" name="cardA">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Title</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>File Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Blocks</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,0">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Memory Card:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cardBPath">
       <property name="currentText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="newCardB">
       <property name="text">
        <string>New...</string>
       </property>
       <property name="icon">
        <iconset resource="resources/resources.qrc">
         <normaloff>:/icons/document-new.png</normaloff>:/icons/document-new.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="openCardB">
       <property name="text">
        <string>Open...</string>
       </property>
       <property name="icon">
        <iconset resource="resources/resources.qrc">
         <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,0,0">
     <item>
      <widget class="QLabel" name="cardBUsage">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importFileToCardB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Import File...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importCardB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Import Card...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveCardB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0,0,0">
     <item>
      <widget class="QLabel" name="cardAUsage">
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importFileToCardA">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Import File...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="importCardA">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Import Card...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveCardA">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Memory Card:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cardAPath"/>
     </item>
     <item>
      <widget class="QToolButton" name="newCardA">
       <property name="text">
        <string>New...</string>
       </property>
       <property name="icon">
        <iconset resource="resources/resources.qrc">
         <normaloff>:/icons/document-new.png</normaloff>:/icons/document-new.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="openCardA">
       <property name="text">
        <string>Open...</string>
       </property>
       <property name="icon">
        <iconset resource="resources/resources.qrc">
         <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <widget class="QTableWidget" name="cardB">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Title</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>File Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Blocks</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="deleteFile">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportFile">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Export File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveLeft">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveRight">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
